test/jdk/java/foreign/TestValueLayouts.java

```diff
@@ -138,7 +146,7 @@ void test(ValueLayout layout,
         assertEquals(layout.carrier(), carrier);
         assertEquals(layout.bitSize(), bitSize);
         assertEquals(layout.order(), ByteOrder.nativeOrder());
-        assertEquals(layout.bitAlignment(), bitAlignment);
+        assertEquals(layout.bitAlignment(), (isAixOS && (layout == JAVA_DOUBLE)) ? (bitSize / 2) : bitAlignment);
```

Comment:
I think this should use `32` instead of `bitSize / 2`:
```java
        assertEquals(layout.bitAlignment(), (isAixOS && (layout == JAVA_DOUBLE)) ? 32 : bitAlignment);
```

Reply:
Agreed and replaced with `32` as suggested above.